library/oneview_appliance_time_and_locale_configuration.py

```diff
@@ -80,8 +82,19 @@ def __init__(self):
         self.resource_client = self.oneview_client.appliance_time_and_locale_configuration
 
     def execute_module(self):
-        resource = self.resource_client.get()
-        return self.resource_present(resource, self.RESOURCE_FACT_NAME)
+        if self.state == 'present':
+            changed, msg, appliance_time_and_locale_configuration = self.__present()
+        return dict(changed=changed, msg=msg, ansible_facts=appliance_time_and_locale_configuration)
+
+    def __present(self):
+        self.current_resource = self.resource_client.get_all()
+        merged_data = self.current_resource.data.copy()
```

Comment:
what if the return value of ```self.resource_client.get_all()``` is None..?
```self.current_resource.data.copy()``` value will fail saying None type has no 'data' attribute. So add the if condition before that.